db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1551,13 +1551,21 @@ Status DBImpl::FlushMemTable(ColumnFamilyData* cfd,
     if (!cfd->mem()->IsEmpty() || !cached_recoverable_state_empty_.load()) {
       s = SwitchMemtable(cfd, &context);
     }
-
     if (s.ok()) {
       if (cfd->imm()->NumNotFlushed() != 0 || !cfd->mem()->IsEmpty() ||
           !cached_recoverable_state_empty_.load()) {
         flush_memtable_id = cfd->imm()->GetLatestMemTableID();
         flush_req.emplace_back(cfd, flush_memtable_id);
       }
+      if (cfd->GetName().compare(kPersistentStatsColumnFamilyName) != 0 &&
```

Comment:
nit: rather than doing the name comparison, can we directly compare the pointer of cfd?